scripts/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex # fail on first error, print commands
+
+SRC_DIR=${SRC_DIR:-$(pwd)}
```

Comment:
i think this is left over, or should be used on line 8?

Reply:
2ddb42e